<ui version="4.0" >
 <class>MainSettingsPage</class>
 <widget class="QWidget" name="MainSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item row="2" column="1" >
      <widget class="QRadioButton" name="openEmptySessionBtn" >
       <property name="text" >
        <string>Open empty session</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QRadioButton" name="showSessionDlgBtn" >
       <property name="text" >
        <string>Show session dialog</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="1" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QRadioButton" name="openLastSessionBtn" >
       <property name="text" >
        <string>Open last saved session</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>On program start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="saveSessionChk" >
     <property name="text" >
      <string>Save session when close</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="syncOpenDlgChk" >
     <property name="text" >
      <string>Sync 'Open' dialog to current doc's dir</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="singleInstanceChk" >
     <property name="toolTip" >
      <string>Use existing instance of application when open files</string>
     </property>
     <property name="text" >
      <string>Open files in existing instance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="makeBackupChk" >
     <property name="text" >
      <string>Make backup copy when save file</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>188</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showSessionDlgBtn</tabstop>
  <tabstop>openLastSessionBtn</tabstop>
  <tabstop>openEmptySessionBtn</tabstop>
  <tabstop>saveSessionChk</tabstop>
  <tabstop>singleInstanceChk</tabstop>
  <tabstop>makeBackupChk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
